tests/cpp_api/CMakeLists.txt

```diff
@@ -25,6 +25,46 @@
 ################################################################################
 cmake_minimum_required (VERSION 3.5)
 
+project(rocal-test)
+
+# make test with CTest
+enable_testing()
+include(CTest)
+
+# ROCM Path
+if(DEFINED ENV{ROCM_PATH})
+  set(ROCM_PATH $ENV{ROCM_PATH} CACHE PATH "Default ROCm installation path")
+elseif(ROCM_PATH)
+  message("-- ${PROJECT_NAME} INFO:ROCM_PATH Set -- ${ROCM_PATH}")
+else()
+  set(ROCM_PATH /opt/rocm CACHE PATH "Default ROCm installation path")
+endif()
+
+# find rocAL
+find_library(ROCAL_LIBRARY NAMES rocal HINTS ${ROCM_PATH}/lib)
+find_path(ROCAL_INCLUDE_DIR NAMES rocal_api.h PATHS ${ROCM_PATH}/include/rocal)
```

Comment:
@kiritigowda Why are we adding only rocal_api.h. We have a few more headers under include/api

Reply:
This is just searching for the header

Reply:
@LakshmiKumar23 - will create an issue for all apps and tests to find rocAL before building.